src/config.rs

```diff
@@ -157,11 +160,130 @@ impl Config for Table {
         }
         array_value
     }
+
+    /// Get a text key and attempt to interpret it into an ANSI style.
+    fn get_as_ansi_style(&self, key: &str) -> Option<ansi_term::Style> {
+        let style_string = self.get_as_str(key)?;
+        parse_style_string(style_string)
+    }
+}
+
+/** Parse a style string which represents an ansi style. Valid tokens in the style
+ string include the following:
+ - 'fg:<color>'    (specifies that the color read should be a foreground color)
+ - 'bg:<color>'    (specifies that the color read should be a background color)
+ - 'underline'
+ - 'bold'
+ - '<color>'        (see the parse_color_string doc for valid color strings)
+*/
+fn parse_style_string(style_string: &str) -> Option<ansi_term::Style> {
+    let tokens = style_string.split_whitespace();
+    let mut style = ansi_term::Style::new();
+
+    // Should we color the foreground? If not, assume we color the background.
```

Comment:
Is the sentence here correct? Seems like we default to foreground if no `fg:`/`bg:` token was found.

Reply:
Ah, I think this is poorly worded.

The intent was to say that "if `col_fg` is `false`, then we're coloring the background", as opposed to "if `col_fg` is false, don't color anything".

Reply:
The other option I had would have been to introduce an Enum for it, but that felt a little overkill for an either-or decision.

Reply:
Thanks. It reads much better now.